Gate failure

<!--
Please add the affected binary name in the title unless multiple binaries are affected, e.g.
[cinder-csi-plugin] Cannot delete PV
For openstack-cloud-controller-manager, you can use [occm] for short.

All the currently maintained binaries are:
* openstack-cloud-controller-manager (occm)
* cinder-csi-plugin
* manila-csi-plugin
* k8s-keystone-auth
* client-keystone-auth
* octavia-ingress-controller
* magnum-auto-healer
* barbican-kms-plugin
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind bug
> /kind feature

**What happened**:

```
I0324 02:31:58.387] go get -u golang.org/x/lint/golint
W0324 02:31:59.673] go: downloading golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
W0324 02:31:59.728] go: downloading golang.org/x/tools v0.1.6-0.20210820212750-d4cc65f0b2ff
W0324 02:32:00.239] go: downloading golang.org/x/tools v0.1.10
W0324 02:32:00.900] go: downloading golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f
W0324 02:32:00.901] go: downloading golang.org/x/sys v0.0.0-20211019181941-9d821ace8654
W0324 02:32:01.063] go: upgraded golang.org/x/net v0.0.0-20210825183410-e898025ed96a => v0.0.0-20211015210444-4f30a5c0130f
W0324 02:32:01.063] go: upgraded golang.org/x/sys v0.0.0-20210831042530-f4d43177bf5e => v0.0.0-20211019181941-9d821ace8654
W0324 02:32:01.063] go: upgraded golang.org/x/tools v0.1.6-0.20210820212750-d4cc65f0b2ff => v0.1.10
W0324 02:32:01.072] make: *** [Makefile:179: lint] Error 1
W0324 02:32:01.074] Traceback (most recent call last):
W0324 02:32:01.075]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 50, in <module>
W0324 02:32:01.075]     main(ARGS.env, ARGS.cmd + ARGS.args)
W0324 02:32:01.075]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 41, in main
W0324 02:32:01.075]     check(*cmd)
W0324 02:32:01.075]   File "/workspace/./test-infra/jenkins/../scenarios/execute.py", line 30, in check
W0324 02:32:01.075]     subprocess.check_call(cmd)
W0324 02:32:01.075]   File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
W0324 02:32:01.075]     raise CalledProcessError(retcode, cmd)
W0324 02:32:01.076] subprocess.CalledProcessError: Command '('make', 'check')' returned non-zero exit status 2
I0324 02:32:01.078] hack/verify-golint.sh
I0324 02:32:01.078] Can not find golint, install with:
I0324 02:32:01.078] go get -u github.com/golang/lint/golint
E0324 02:32:01.078] Command failed
```

in https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/cloud-provider-openstack/1814/pull-cloud-provider-openstack-check/1506820645856481280/build-log.txt

**What you expected to happen**:


**How to reproduce it**:


**Anything else we need to know?**:


**Environment**:
- openstack-cloud-controller-manager(or other related binary) version:
- OpenStack version:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -175,7 +175,7 @@ fmt:
 lint:
 ifndef HAS_LINT
 	echo "installing lint"
-	go get -u golang.org/x/lint/golint
+	go install golang.org/x/lint/golint@latest
 endif
 	hack/verify-golint.sh
 

--- a/hack/verify-golint.sh
+++ b/hack/verify-golint.sh
@@ -24,7 +24,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 if ! which golint > /dev/null; then
   echo 'Can not find golint, install with:'
-  echo 'go get -u github.com/golang/lint/golint'
+  echo 'go install golang.org/x/lint/golint@latest'
   exit 1
 fi
 

